tutorials/remote-staking-xcm.md

```diff
@@ -0,0 +1,118 @@
+
+Now we’re ready for the fun part! To construct the XCM message that will remotely stake funds on the Moonbase Alpha parachain, take the following steps:
+
+1. Double check that `V2` is selected as the XCM version
+2. Ensure `WithdrawAsset` is selected as the first XCM instruction
+3. Select `Concrete` as the XcmV1MultiAsset
+4. Specify a parents value of `0`
+5. Specify `X1` because we need one piece of information to identify the asset 
+6. Select `PalletInstance`
+7. Specify PalletInstance `3` - this is the `balances` pallet. In other words, we are identifying the native asset of the Moonbase parachain, which is the DEV token 
+8. Specify fungibility as `fungible`. DEV, like GLMR and MOVR, is a fungible asset 
+9. Specify `100000000000000000` Wei as the fungible amount. This is equivalent to `0.1` DEV and it is a safe overestimate of the fees that will actually need to be paid. In a production environment, we would consider including XCM instructions to refund unused fees, but for the sake of simplicity, that is not included in this tutorial
+10. Now, we’ll begin with our second XCM instruction. Select `buyExecution` from the dropdown. We are buying execution with the same asset (DEV) that we withdrew in the prior steps, so the following steps will be familiar 
+11. Select `Concrete` as the XcmV1MultiAsset
+12. Specify a parents value of `0`
+13. Specify `X1` because we need one piece of information to identify the asset
+14. Select `PalletInstance`
+15. Specify PalletInstance `3` - this is the balances pallet. In other words, we are identifying the native asset of the Moonbase parachain, which is the DEV token
+16. Specify fungibility as `fungible`. DEV, like GLMR and MOVR, is a fungible asset  
+17. Specify `100000000000000000` Wei as the fungible amount. This is equivalent to `0.1` DEV
+18. Select `Unlimited` for the weightLimit
+19. For our 3rd and most crucial XCM instruction, choose the `Transact` instruction
+20. Specify an origin of the `SovereignAccount`
+21. For requireWeightAtMost, specify `40000000000`. This is likely more than strictly necessary, but it is fine to overestimate
+22. Finally, paste the encoded call data of the staking operation that you prepared on Moonbase Alpha in Step 7 of the first section of this tutorial
+23. Press **Submit** and enter your password in the resulting popup to confirm the transaction
+
+![Assembling the complete XCM message](/images/tutorials/remote-staking-via-xcm/xcm-stake-6.png)
+
+And that’s it! To verify that your delegation was successful, you can visit [Subscan](https://moonbase.subscan.io/){target=_blank} to check your staking balance. Be advised that it may take a few moments before your staking balance is visible on Subscan. Additionally, be aware that you will not be able to see this staking operation on Moonscan, because we initiated the delegation action directly via the parachainStaking pallet (on the substrate side) rather than through the staking precompile (on the EVM). 
```

Comment:
```suggestion
And that’s it! To verify that your delegation was successful, you can visit [Subscan](https://moonbase.subscan.io/){target=_blank} to check your staking balance. Be advised that it may take a few moments before your staking balance is visible on Subscan. Additionally, be aware that you will not be able to see this staking operation on Moonscan, because we initiated the delegation action directly via the parachain staking pallet (on the substrate side) rather than through the staking precompile (on the EVM). 
```